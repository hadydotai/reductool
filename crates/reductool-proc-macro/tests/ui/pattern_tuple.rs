use reductool_proc_macro::aitool;

#[aitool]
fn f((a, b): (i32, i32)) {}

fn main() {}
