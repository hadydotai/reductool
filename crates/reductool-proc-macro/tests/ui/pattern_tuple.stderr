error: unsupported parameter pattern. expected a simple identifier like `arg: T`.
       Examples of unsupported patterns: `(_: T)`, `(a, b): (T, U)`, `S { x, y }: S`.
 --> tests/ui/pattern_tuple.rs:4:6
  |
4 | fn f((a, b): (i32, i32)) {}
  |      ^^^^^^
